clients/mist_client.go

```diff
@@ -17,19 +19,26 @@ type MistAPIClient interface {
 	AddStream(streamName, sourceUrl string) error
 	PushStart(streamName, targetURL string) error
 	DeleteStream(streamName string) error
-	AddTrigger(streamName, triggerName string) error
-	DeleteTrigger(streamName, triggerName string) error
+	AddTrigger(streamName []string, triggerName string, sync bool) error
```

Comment:
1. Why do we need to change this `string` into `[]string`?
2. Do we still need all these functions: `AddStream`, `PushStart`, etc. I don't see their usage anymore. Maybe we can clean them up?

Reply:
1. We want our triggers to apply to all streams on the node, which is represented in Mist as an empty array; thus the signature change.
2. Some of them aren't used right now, such as `PushStart`, but they _will_ be used as part of fixing multistreaming. So I'd prefer not to delete them now and bring them back later.

More broadly, there are two Mist clients right now -- one in the `mapic` directory and one in the `clients` directory. These need to get consolidated, I'll look into that today.

Reply:
Yeah, but I also wonder if we actually use all these functions anywhere? Like `PushStart()`?

Reply:
Oops, put the other comment in the wrong place. We use PushStart now that I consolidated the two Mist clients - mapic uses it to start multistreaming